spark/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/raster/RasterFunctions.scala

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
I suggest we should change the old implementation in place, instead of creating a new .Scala file for these special functions

Reply:
The reason why I create new scala wrappers is that if we change the Java function (RS_Metadata), since they are under the sedona-common module, it does not have dependency for spark. However, in order to return struct, we need 
```
import org.apache.spark.sql.types.{AbstractDataType, BooleanType, DataType, DoubleType, IntegerType, StructField, StructType}
```

I think it is not appropriate to introduce spark dependencies to the sedona-common module. The wrapper is also consistent with some other RS functions in PixelFunctions for example.